### ISSUE TITLE
Expose more of minizip's info struct

### ISSUE THREAD

**Issue by Togtja** [sebastiandev/zipper#94]
Do you know if the CRC of the extracted zipfiles get checked when extracting using Minizip/Zipper to extract?

If it does not I think we should expose the CRC in the `ZipEntry`, and maybe create some functions that unzips and checks the CRC

**Comment by Togtja** [comment]
After digging into minizip and writing some test code. I figured out that minizip does test the CRC when unzipping. However, it throws a generic "UNZ_BADZIPFILE" error, meaning it can be hard to tell exactly why it failed.

I still belive that exposing the CRC, and potetially other minizip file info, like the CRC and Compression method, would be nice for those of us who want to do some more checking and validating.
Maybe also version and flags from the minizip `unz_file_info64` struct

(I have editied the Title to reflect this)
Original Title: `CRC and Minizip/Zipper`
